src/github/issue.ts

```diff
@@ -1,23 +1,16 @@
-import { CustomContext } from "./middleware";
 import JiraClient from "../models/jira-client";
 import { emitWebhookProcessingTimeMetrics } from "../util/webhooks";
 import { booleanFlag, BooleanFlags } from "../config/feature-flags";
+import { CustomContext } from "./middleware";
 
-export default async (
-	context: CustomContext,
-	_: JiraClient,
-	util
-): Promise<void> => {
+export default async (context: CustomContext, _: JiraClient, util): Promise<void> => {
 	const { issue } = context.payload;
 
 	let linkifiedBody;
 	try {
 		linkifiedBody = await util.unfurl(issue.body);
 		if (!linkifiedBody) {
-			context.log(
-				{ noop: "no_linkified_body_issue" },
-				"Halting further execution for issue since linkifiedBody is empty"
-			);
+			context.log("Halting further execution for issue since linkifiedBody is empty");
```

Comment:
Why remove the `noop` field in the log event? It might be handy to filter for noop reasons. Then again, I don't care enough about this field to talk too much about it...

Reply:
`noop` isn't a standard for us and it was a relic of github's way of doing things.  It also had no corrolation between messages and we can just as easily just filter by the message itself.